src/display.rs

```diff
@@ -207,3 +209,155 @@ mod tests {
                     \"foo.bar\" = 2\n");
     }
 }
+
+
+
+
+/// Encodes an encodable value into a "pretty" TOML string.
+///
+/// This function expects the type given to represent a TOML table in some form.
+/// If encoding encounters an error, then this function will fail the task.
+/// 
+/// "pretty" means the following features of the output are changed:
+/// - strings with newlines characters (`\n`) will use the `'''` form
+///     and span multiple lines
+pub fn encode_str_pretty<'a>(tbl: TomlTable) -> Result<StdString, fmt::Error> {
```

Comment:
With something like that we could also support builder-style configuration on `Pretty` itself, adding support for various options over time.